src/__tests__/check_in.test.ts

```diff
@@ -493,13 +492,63 @@ describe("checkIn.Cron", () => {
     })
   })
 
+  describe("Appsignal.checkIn.heartbeat()", () => {
+    beforeAll(() => {
+      heartbeatInterval.set(() => 20)
+    })
+
+    afterEach(() => {
+      killContinuousHeartbeats()
+    })
+
+    afterAll(() => {
+      heartbeatInterval.reset()
+    })
+
+    it("sends a heartbeat event", async () => {
+      heartbeat("test-heartbeat")
+
+      await scheduler.shutdown()
+
+      expect(requests).toHaveLength(1)
+      expectEvents(requests[0], [
+        {
+          identifier: "test-heartbeat",
+          check_in_type: "heartbeat"
+        }
+      ])
+    })
+
+    describe("with the `continuous: true` option", () => {
+      it("sends heartbeat events continuously", async () => {
+        debounceTime.set(() => 20)
+
+        heartbeat("test-heartbeat", { continuous: true })
+
+        await sleep(60)
```

Comment:
Could we speed this up by mocking the sleep time value in the scheduler?

Reply:
We do that ([for the heartbeat here](https://github.com/appsignal/appsignal-nodejs/pull/1123/files/839073e8c3a13956e7ec5019039095e15a7cf402#diff-2ef3a10248c9f7c4a782cdefe5f5f5ccbef615d8a55e2303a170bc8e47279f82R496-R498) and [for the scheduler's debounce here](https://github.com/appsignal/appsignal-nodejs/pull/1123/files/839073e8c3a13956e7ec5019039095e15a7cf402#diff-2ef3a10248c9f7c4a782cdefe5f5f5ccbef615d8a55e2303a170bc8e47279f82R524))

Note that this is 60 milliseconds, not 60 seconds!